### PR TITLE
LD-531 - Adds correct margin/padding to last carousel image

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/LD-531

### CODE REVIEW

File: src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx [hunk 1]
```diff
@@ -40,7 +40,12 @@ export class Carousel extends React.Component<Props> {
     const { sources } = this.props
 
     return sources.map((source, i) => (
-      <Flex key={this.keyForSource(source)} mr={1} alignItems="flex-start">
+      <Flex
+        key={this.keyForSource(source)}
+        mr={i === sources.length - 1 ? 2 : 1}
+        pr={i === sources.length - 1 ? 2 : 0}
```
Comment (ashfurrow) [inline src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx]: We repeat this logic twice: `i === sources.length - 1`. Maybe we could put it in a local var? It'd make it easier to read.

File: storybook/storyLoader.js [hunk 2]
```diff
@@ -1,99 +1,102 @@
...
+  '../src/lib/Scenes/Settings/__stories__/Settings.story',
+  '../src/lib/Scenes/Show/__stories__/Show.story',
+  
+];
 
 module.exports = {
   loadStories,
   stories,
-}
+};
```
Comment (ashfurrow) [inline storybook/storyLoader.js]: I'm seeing these changes locally to, let's commit  them here 👍

File: src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx (hunk 1 above)
Comment (kierangillen) [reply src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx]: Makes sense, just updated

**Comment by ashfurrow** [comment]
Our TypeScript error-parsing is broken (see @orta's comment on #1553), but here's the problem:

> src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx(55,3): error TS1005: ',' expected.

**Pull request merged**